cfn-resources/autogen/Makefile

```diff
@@ -0,0 +1,29 @@
+.PHONY: submit test
+tags=logging callback metrics scheduler
+cgo=0
```

Comment:
[q] Could you move the `autogen` folder outside `cfn-resource`? I think that only actual resources should live in that folder where this is not

Reply:
Hi andrea, the go.mod is in the cfn-resource folder, all the go project is based there, moving the folder outside, will need to setup a new go.mod and change most files and urls.

Reply:
oh I see, thanks for the clarification